process/block/blockProcessor.go

```diff
@@ -80,14 +81,49 @@ func (bp *blockProcessor) PrepareBlockForDB(
 		StateRootHash:         hex.EncodeToString(header.GetRootHash()),
 		PrevHash:              hex.EncodeToString(header.GetPrevHash()),
 		SearchOrder:           computeBlockSearchOrder(header),
-		AccumulatedFees:       header.GetAccumulatedFees().String(),
-		DeveloperFees:         header.GetDeveloperFees().String(),
 		EpochStartBlock:       header.IsStartOfEpochBlock(),
 	}
 
+	if header.GetAccumulatedFees() != nil {
```

Comment:
please extract header.GetAccumulatedFees() in a variable as to not call the method twice. Valid also for header.GetDeveloperFees()

Reply:
done.